gpkit/nomials/nomial_math.py

```diff
@@ -509,10 +503,10 @@ def as_posyslt1(self, substitutions=None):
         for posy in posys:
             _, exps, cs, subs = substitution(posy, substitutions)
             self._last_used_substitutions = subs
-            exps, cs = self._simplify_posy_ineq(exps, cs)
+            exps, cs, pmap = simplify_exps_and_cs(exps, cs, return_map=True)
+            exps, cs, pmap = self._simplify_posy_ineq(exps, cs, pmap)
```

Comment:
There's a lot going on in the four lines above (substitution, and two simplification steps). Can you add an inline comment to clarify what the two different simplification steps are? Or perhaps make the method names more descriptive?

It's particularly interesting to me that the two simplification steps switched order in this PR -- what's up with that?